doc/api/crypto.md

```diff
@@ -1138,8 +1141,18 @@ changes:
 -->
 * {string}
 
-For asymmetric keys, this property represents the type of the embedded key
-(`'rsa'`, `'dsa'`, `'ec'`, `'ed25519'`, `'ed448'`, `'x25519'` or `'x448'`).
+For asymmetric keys, this property represents the type of the embedded key. This
+version of Node.js supports the following asymmetric key types:
+
+* `'rsa'` (OID 1.2.840.113549.1.1.1)
+* `'rsa-pss'` (OID 1.2.840.113549.1.1.10)
```

Comment:
I don't understand why OpenSSL calls this a "key type". Its the same ASN.1:

       Note: It is not possible to restrict the use of a key to a set of
      algorithms (i.e., RSASSA-PSS and RSAES-OAEP).

      Regardless of the object identifier used, the RSA public key is
      encoded in the same manner in the subject public key information.

- RFC4055

Wouldn't there also be an 'rsa-oaep' key type, as well? Seems weird to me.